src/components/step-tooltip/styles.scss

```diff
@@ -0,0 +1,154 @@
+.StepTooltip {
```

Comment:
Ideally use the naming style we use in the rest of the project. For class names, use kebab-case. So this one would be `step-tooltip`